lib/taskana-core/src/test/java/acceptance/task/claim/SetOwnerAccTest.java

```diff
@@ -50,6 +50,25 @@ void testSetOwnerAndSubsequentClaimSucceeds() throws Exception {
     assertThat(taskClaimed.getOwner()).isEqualTo("user-1-2");
   }
 
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void testSetOwnerOfReadyForReviewTaskAndSubsequentClaimSucceeds() throws Exception {
```

Comment:
this naming is not according to our guidelines :wink:

Reply:
I thought I stick to the naming of the tests in this class and keep the refactoring for a seperate ticket ^^